com.unity.render-pipelines.universal/Runtime/XR/XRPass.cs

```diff
@@ -406,6 +406,12 @@ internal void EndCamera(CommandBuffer cmd, CameraData cameraData)
 
         internal void RenderOcclusionMesh(CommandBuffer cmd)
         {
+
+        #if DEVELOPMENT_BUILD || UNITY_EDITOR
+            if (XRGraphicsAutomatedTests.enabled && XRGraphicsAutomatedTests.running)
```

Comment:
Is there a good reason why we disable occlusion mesh when automation is in use?

Reply:
By default, the occlusion mesh from MockHMD will show up because we're using the full viewport 1:1 during tests.
The alternative would be to add a right-eye only mirror view mode without occlusion mesh to MockHMD somehow.

Reply:
Makes sense. Thanks!